crates/router/src/connector/checkout/transformers.rs

```diff
@@ -882,7 +996,11 @@ pub struct CheckoutWebhookData {
     pub id: String,
     pub payment_id: Option<String>,
     pub action_id: Option<String>,
+    pub reference: Option<String>,
     pub amount: i32,
+    pub balances: Option<Balances>,
```

Comment:
Was this added because of try_from for PaymentsResponse?

Reply:
yes